tests/test_devices.py

```diff
@@ -60,6 +60,36 @@ def test_device_constructor_with_config(client):
     assert device.inputs_wait == {}
 
 
+def test_item_query_without_status(alarm_device):
+    """Verify that querying items without specifying a status works correctly"""
+    alarm_device.connect("username", "password")
+    # Test
+    alarm_device.update()
+    assert dict(alarm_device.items(q.SECTORS)) == alarm_device._inventory.get(q.SECTORS)
```

Comment:
In tests always be explicit, so don't check with `alarm_device._inventory.get(q.SECTORS)` but instead write the data structure (so the dict) you expect. This is important because if you keep the test that way, I can write a wrong code that will make the test pass anyway.